templates/base.html

```diff
@@ -10,12 +10,12 @@
     <meta name="color-scheme" content="light dark" />
 
     <title>{% block title %} {% endblock title %}</title>
-
     <link rel="alternate" type="application/rss+xml" title="RSS" href="/rss.xml" />
     <link rel="icon" type="image/png" href="{{ get_url(path="images/favicon.svg") }}" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@picocss/pico@2.0.6/css/pico.min.css" />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/fork-awesome@1.2.0/css/fork-awesome.min.css" integrity="sha256-XoaMnoYC5TH6/+ihMEnospgm0J1PM/nioxbOUdnM8HY=" crossorigin="anonymous">
     <link rel="stylesheet" href="{{ get_url(path="css/style.css") }}" />
+    <script data-goatcounter="https://limeleaf.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
```

Comment:
I would move this to right before the `</body>` so it doesn't block loading the page itself. Most JS scripts these days end up at the bottom of the page for that reason to ensure the DOM is fully loaded before any scripts that could use elements in the DOM.